pkg/sbom/nodelist.go

```diff
@@ -266,6 +266,27 @@ func copyEdgeList(original []*Edge) []*Edge {
 	return nodeCopy
 }
 
+// Copy returns a duplicate of the NodeList.
+func (nl *NodeList) Copy() *NodeList {
+	nlo := &NodeList{
+		// 2DO not sure what to do.
+		// state:         nl.state,
+		// sizeCache:     nl.sizeCache,
+		// unknownFields: nl.unknownFields,
```

Comment:
We've dropped thse from other copy funcs
```suggestion
```

Reply:
Resolved by merging in main back to branch.